src/Modules/Registration/RegistrationTextInfoComponent.js

```diff
@@ -1,11 +1,13 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useEffect, useState } from 'react';
 import TagManager from 'react-gtm-module'
 
 
 import RegistrationHeaderComponent from './RegistrationHeaderComponent';
 import EventCardComponent from '../Events/EventCardComponent';
+import BackButtonComponent from '../General/BackButtonComponent';
 
 const RegistrationTextInfoComponent = ({event, onRegisterNow}) => {
+const isFBLoggedIn = localStorage.getItem("isFBLoggedIn");
```

Comment:
In order to get back the real boolean values, we can use the JSON. parse() method.
here it returns value as "false" but not `false`

Reply:
updated in this commit - https://github.com/midohiofoodbank/freshtrak-client/pull/188/commits/4d07e048e92cee77474cf67b1a71ba4c11e42f04